### PR TITLE
Fixes #262: Added Support for Apple Silicon M1

### LINKED ISSUE #262: Build for Apple Silicon (macos-arm64)
Plugin install fails on ARM-based Macs. I was able to build from source and copy the executable to my helm plugins directory.

I am using helm 3.5.2 and helmfile 0.138 from Homebrew.

```
nick@macbook-air ~ % helm plugin install https://github.com/databus23/helm-diff
No prebuild binary for macos-arm64.
Failed to install helm-diff
	For support, go to https://github.com/databus23/helm-diff.
Error: plugin install hook for "diff" exited with error
```

### PR DESCRIPTION
**Pull request by f3lan**
I simply added the architecture to the supported list. I tested this on
my Mac and it worked.

### CODE REVIEW

**Review by abh — APPROVED** [review]
looks good to me ...

**Comment by dmerrick** [comment]
I'm interested in this!

**Review by dmerrick — CHANGES_REQUESTED** [review]

File: install-binary.sh [hunk 1]
```diff
@@ -55,7 +55,7 @@ initOS() {
 # verifySupported checks that the os/arch combination is supported for
 # binary builds.
 verifySupported() {
-  supported="linux-amd64\nfreebsd-amd64\nmacos-amd64\nwindows-amd64"
+  supported="linux-amd64\nfreebsd-amd64\nmacos-amd64\macos-arm64\nwindows-amd64"
```
[the remark this answers is not in the document]
Comment (f3lan) [reply install-binary.sh]: hi @dmerrick That is completely true. Im sorry for this bad mistake. I corrected this with my last commit.

**Comment by yseymour** [comment]
This looks to have been merged in #294.

**Comment by Hammond95** [comment]
I think this can be closed since #294 has been merged.

**Comment by mumoshu** [comment]
Yep. This can be closed and it should install fine on M1 mac now! Thanks for your patience and support everyone

**Pull request closed without merging**